src/main/java/org/folio/fqm/migration/MigrationStrategyRepository.java

```diff
@@ -3,14 +3,15 @@
 import java.util.Collections;
 import java.util.List;
 import org.folio.fqm.migration.strategies.V0POCMigration;
+import org.folio.fqm.migration.strategies.V1ModeOfIssuanceConsolidation;
 import org.springframework.stereotype.Component;
 
 @Component
 public class MigrationStrategyRepository {
 
   // prevent re-initialization on each call
   private static final List<MigrationStrategy> MIGRATION_STRATEGIES = Collections.unmodifiableList(
-    List.of(new V0POCMigration())
+    List.of(new V0POCMigration(), new V1ModeOfIssuanceConsolidation())
   );
```

Comment:
Tiniest of nits that I wish I'd seen in an earlier PR: IIRC, the list returned from `List.of()` is already immutable, so there's no need to wrap it with `Collection.unmodifiableList()`.